srcpkgs/keybase/template

```diff
@@ -1,21 +1,22 @@
 # Template file for 'keybase'
 pkgname=keybase
-version=5.8.1
-revision=5
+version=6.4.0
+revision=1
 build_style=go
-go_import_path="github.com/keybase/client"
-go_package="${go_import_path}/go/keybase
-${go_import_path}/go/kbfs/kbfsfuse
-${go_import_path}/go/kbfs/kbfsgit/git-remote-keybase
-${go_import_path}/go/kbfs/kbfstool ${go_import_path}/go/kbfs/redirector"
+build_wrksrc=go
+go_import_path="github.com/keybase/client/go"
+go_package="${go_import_path}/keybase
+${go_import_path}/kbfs/kbfsfuse
+${go_import_path}/kbfs/kbfsgit/git-remote-keybase
+${go_import_path}/kbfs/kbfstool ${go_import_path}/kbfs/redirector"
```

Comment:
I'm not sure why these changes were necessary

Reply:
According to @pondichys this is the go import path in go.mod file

Reply:
Sorry, to clarify: the go.mod file import path appends /go sub-directory.

Reply:
Ah sure.